classes/04class/iac/artifacts/example2/vpc.tf

```diff
@@ -0,0 +1,125 @@
+resource "aws_vpc" "devops_academy" {
+  cidr_block = "192.168.0.0/16"
+  tags = {
+    Name = "DevopsAcademy"
+  }
+}
+resource "aws_subnet" "subnet_private_a" {
+  vpc_id            = "${aws_vpc.devops_academy.id}"
```

Comment:
tf 0.12+ does not require the `"${}"` anymore on var interpolation if strings are not present. i.e:
` vpc_id            = aws_vpc.devops_academy.id`

Reply:
please update other references as well

Reply:
this wasn't supposed to go to the repo.. lol

Reply:
if it is useful for the class, I don't see why not 🤷‍♂️

Reply:
it wasn't created with that intention, so it didn't used best practices neither tf .12 syntax.
it's also the entire answer for the exercise 02, so that's the reason I didn't included in the repo when I've created it.
Don;t mind adding it as long as the code is written properly to be added to the repo.

Reply:
OK, let's version any resource used in class for future instructors.

Changes requested done 👍